packages/fx-core/tests/core/other.test.ts

```diff
@@ -93,4 +113,51 @@ describe("Other test case", () => {
     assert.isTrue(error.name === "FetchSampleError");
     assert.isTrue(error.message === "Failed to download sample app");
   });
-});
\ No newline at end of file
+
+  it("tools: isFeatureFlagEnabled should return true when related environment variable is set to 1 or true", () => {
+    const featureFlagName = "FEATURE_FLAG_UNIT_TEST";
+    let restore = mockedEnv({
+      [featureFlagName]: "1",
+    });
+    assert.isTrue(isFeatureFlagEnabled(featureFlagName));
+
+    restore = mockedEnv({
+      [featureFlagName]: "true",
+    });
+    assert.isTrue(isFeatureFlagEnabled(featureFlagName));
+
+    restore = mockedEnv({
+      [featureFlagName]: "TruE", // should allow some characters be upper case
+    });
+    assert.isTrue(isFeatureFlagEnabled(featureFlagName));
+
+    restore();
```

Comment:
seems called restore() will restore featureFlagName as "true", is this with expectations?
maybe restore need to called before mockedEnv?

Reply:
Thanks, seems I missed something when experimenting the behavior of mockedEnv. Will double confirm and update the test cases accordingly.

Reply:
resolved